plugins/nginx-vhosts/subcommands/migrate

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/config/functions"
+source "$PLUGIN_AVAILABLE_PATH/nginx-vhosts/functions"
+
+nginx_migrate_cmd() {
+  declare desc="converts DOKKU_NGINX_* to DOKKI_PROXY_*"
+  local cmd="nginx:migrate"
+  migrate_nginx_to_proxy
```

Comment:
I really don't want this to be a thing. For anyone installing from source, the makefile will trigger the install hook, and packaged installs also trigger it as well.